doc/devel/documenting_mpl.rst

```diff
@@ -54,22 +54,19 @@ The documentation sources are found in the :file:`doc/` directory in
 the trunk.  To build the users guide in html format, cd into
 :file:`doc/` and do::
 
-  python make.py html
+  make html
 
-or::
 
-  ./make.py html
-
-There are many other flags you can pass to ``make.py``, and you can see the
-full list inside that file. Here are two useful ones:
+There are many other options you can pass to ``make``, and you can see the
+full list inside that file. Here are three useful ones:
 
+* ``html-noplots`` will build the documentation without writing the gallery
```

Comment:
Likewise would document as `make SPHINXOPTS=-Dplot_gallery=0 html`.

One advantage is that this can easily be combined with (dis)allowsphinxwarnings (`make SPHINXOPTS='Dplot_gallery=0 -W' html`)